cpp/src/io/json/json_tree.cu

```diff
@@ -36,6 +36,7 @@
 #include <cub/device/device_radix_sort.cuh>
 
 #include <cuco/static_map.cuh>
```

Comment:
```suggestion
```

Reply:
There is another place in this file where `static_map` is used.

Reply:
Ah OK. Resovling this comment.